scripts/get-secrets.sh

```diff
@@ -11,9 +11,9 @@ function main {
     > $env_secret_file
   fi
 
-  secrets=$(aws ssm --region eu-west-1 get-parameters-by-path \
-    --path "/k8s/$1/" --with-decryption --recursive \
-    --parameter-filters Key=Label,Option=Equals,Values=dev \
+  secrets=$(aws ssm --region eu-west-1 get-parameters-by-path /
```

Comment:
definitely not - these backslashes are cancelling newlines

Reply:
Will be fixed in next commit.